int/api/webcreator.go

```diff
@@ -45,12 +45,35 @@ func PrepareForWebsiteHandler(params operations.WebsiteCreatorPrepareParams) mid
 				})
 	}
 
+	archive, err := ioutil.ReadAll(params.Zipfile)
```

Comment:
ioutil needs to be changed to io or os. Also the package import needs to be updated to match the new package.
Source : https://pkg.go.dev/io/ioutil